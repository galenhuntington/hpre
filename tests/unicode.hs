{-# LANGUAGE LambdaCase, UnicodeSyntax #-}

str1 :: String
〃  = "test1"

isPosU ∷ Int → Bool
”   = \case
   x | x > 0 → True
     |       → False

