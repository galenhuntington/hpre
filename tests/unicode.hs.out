{-# LANGUAGE LambdaCase, UnicodeSyntax #-}

str1 :: String
str1{-#COLUMN 4#-}= "test1"

isPosU ∷ Int → Bool
isPosU{-#COLUMN 5#-}= \case
   x | x > 0 → True
     | True  → False


